tests/scr/testContainerInit

```diff
@@ -243,7 +243,7 @@ function build_sysbox() {
 		make sysbox-debug-local --no-print-directory && make install
 	else
 		echo "Building sysbox ..."
-		make sysbox-local --no-print-directory && make install
+		make sysbox-static-local --no-print-directory && make install
```

Comment:
In that case, we need to replace `sysbox-debug-local` with its `static` equivalent too. We can do it at a later time though.

Reply:
True, let's do that later since it's a larger but less consequential change.